src/mkfs.famfs.c

```diff
@@ -80,7 +83,13 @@ main(int argc, char *argv[])
 		case 'f':
 			force++;
 			break;
-
+		case 'l':
+			loglen = strtoull(optarg, 0, 0);
+			if (loglen & 1) {
```

Comment:
Shouldn't this be (loglen & (loglen - 1)) ? assuming we are checking if the value is power of 2.